src/d3a/models/area/__init__.py

```diff
@@ -209,7 +209,7 @@ def get_grid_fee(self):
         grid_fee_type = self.config.grid_fee_type \
             if self.config is not None \
             else ConstSettings.IAASettings.GRID_FEE_TYPE
-        return self.get_path_to_root_fees() if grid_fee_type == 1 else 0
+        return self.grid_fee_constant if grid_fee_type == 1 else self.grid_fee_percentage
```

Comment:
```suggestion
        return self.grid_fee_constant if grid_fee_type == 1 else 0.
```

Reply:
@faizan2590 Can you explain please?

Reply:
It was decided to only return constant grid fee and ignore the percentage based fee: https://github.com/gridsingularity/d3a/pull/898#discussion_r503948590

Reply:
hmm, ok, but then the whole function is useless actually, then we can just return the `self.grid_fee_constant` without getting the grid_fee type.
The user can not set up the percentage pased grid fee anyway. So I propose to deal with this in a separate "deprecate percentage grid fee story" What do you think. Also tagging @spyrostz

Reply:
I would actually prefer to not touch the percentage grid fee, since first of all it is exposed in the UI as far as I recall

Reply:
> So I propose to deal with this in a separate "deprecate percentage grid fee story" 

Yes please make a separate ticket for deprecating it. Thanks!

Reply:
https://github.com/gridsingularity/d3a/pull/906#discussion_r506587199